### ISSUE TITLE
TypeError: 'odict_items' object is not subscriptable

### ISSUE THREAD

**Issue by job** [job/irrtree#12]
```
job@bench irrtree$ irrtree AS-SNIJDERS
Processed: 39 objects (Elapsed Time: 0:00:05)                                                                                                                                IRRTree (1.1.3) report for 'AS-SNIJDERS' (IPv4), using rr.ntt.net at 2020-12-18 16:57
Traceback (most recent call last):
  File "/usr/local/bin/irrtree", line 11, in <module>
    load_entry_point('irrtree', 'console_scripts', 'irrtree')()
  File "/home/job/source/irrtree/irrtree/cli.py", line 275, in main
    process(server.irr_host, server.afi, db, query_object, server.search)
  File "/home/job/source/irrtree/irrtree/cli.py", line 178, in process
    print(tr(tree))
  File "/usr/local/lib/python3.8/site-packages/asciitree/__init__.py", line 48, in __call__
    return '\n'.join(self.render(self.traverse.get_root(tree)))
  File "/usr/local/lib/python3.8/site-packages/asciitree/traversal.py", line 29, in get_root
    return tree.items()[0]
TypeError: 'odict_items' object is not subscriptable
```

**Comment by job** [comment]
Seems asciitree might not be entirely py3 compatible.

We might need an alternative library or fork + patch the old 0.3.3 one?

**Comment by job** [comment]
@jnicholson56 what do you think

**Comment by jnicholson56** [comment]
asciitree error. Seems the package may be abandoned. I found the fix in a fork of the project.

![Screen Shot 2020-12-18 at 11 09 50](https://user-images.githubusercontent.com/35084818/102641639-0300a700-4122-11eb-9973-cdfe14a00767.png)

**Comment by job** [comment]
Fixed by pinning to 0.3.3

Thanks!
